### PR TITLE
Add one case to cover set blkiotune values, then start VM --bz 1645471

### PR DESCRIPTION
**Pull request by chunfuwen**
XX-130684

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
cleaning libvirtd logs...
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virsh.blkiotune.negative.set_blkio_parameter.shutoff_guest.change_device_weights.options.none.vm_fail_start

File: libvirt/tests/src/virsh_cmd/domain/virsh_blkiotune.py [hunk 1]
```diff
@@ -207,6 +207,14 @@ def set_blkio_parameter(test, params, cgstop):
                           " blkio.weight and blkio.weight_device"
                           " value from cgroup blkio controller")
 
+    # Start Vm if needed
+    validate_vm_not_start = "yes" == params.get("validate_vm_not_start", "no")
+    if validate_vm_not_start:
+        result = virsh.start(vm_name, debug=True)
+        vm_not_start_error_msg = params.get("vm_not_start_error_msg")
+        if vm_not_start_error_msg not in result.stderr_text:
```
Comment (kylazhang) [inline libvirt/tests/src/virsh_cmd/domain/virsh_blkiotune.py:215]: Do you need to check if vm_not_start_error_msg is not None?

Comment (chunfuwen) [reply libvirt/tests/src/virsh_cmd/domain/virsh_blkiotune.py:215]: @kylazhang 
vm_not_start_error_msg will be extracted under the condition that  under validate_vm_not_start is True, and vm_not_start_error_msg was initialized in cfg file if  validate_vm_not_start is True.
So it is not necessary that check if vm_not_start_error_msg is not None

**Review by dzhengfy — APPROVED** [review]
LGTM

**Review by kylazhang — APPROVED** [review]

**Pull request merged**